core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -92,13 +92,7 @@ export class ChangeListService implements OnInit {
     private loaderService: LoaderService,
     private loggerService: LoggerService,
     private internetConnectivityService: InternetConnectivityService,
-  ) {}
-
-  ngOnInit(): void {
-    this.loaderService.onLoadingMessageChange.subscribe(
-      (message: string) => this.loadingMessage = message
-    );
-
+  ) {
```

Comment:
@Radesh-kumar, Can you please add a comment here that we have added this in the constructor because ngOnInIt doesn't work in service?
I forgot to add a comment when I added my code. Sorry for the trouble.

Reply:
There seems to be some confusion here. If ngOnInit does not work per @atpug22's comment, how does it work now (this PR still uses it)?

Reply:
@kevintab95 Yup, ngOnInIt doesn't work in angular. [Reference](https://github.com/angular/angular/issues/23235).
I tested it manually, and I guess we'll need to remove ngOnInIt completely and then move the loader service subscription here.
@Radesh-kumar, Can you please check this?

Reply:
done

Reply:
@Radesh-kumar, Can you please add the comment too as mentioned above?

Reply:
done